1.py

```diff
@@ -0,0 +1,70 @@
+# Дорабатываем задачу 4. Добавьте регистрацию возможных ошибок. 
+#Добавьте возможность запуска из командной строки с использованием библиотеки 
+#argparse
+
+
+import logging
+from datetime import datetime, date
+from collections import namedtuple
+import argparse
+
+parser = argparse.ArgumentParser(description='Get date') 
+parser.add_argument('text', metavar='text', type=str, nargs=1, help='') 
+if __name__ == '__main__':args = parser.parse_args() 
+logging.basicConfig(filename='task_1.log', encoding='utf-8', level=logging.INFO)
+logger = logging.getLogger('task_1.log')
+
+MONTHS = {
+    'января': 1,
+    'февраля': 2,
+    'марта': 3,
+    'апреля': 4,
+    'мая': 5,
+    'июня': 6,
+    'июля': 7,
+    'августа': 8,
+    'сентября': 9,
+    'октября': 10,
+    'ноября': 11,
+    'декабря': 12
+}
+WEEKDAYS = {
+    'понедельник': 0,
+    'вторник': 1,
+    'среда': 2,
+    'четверг': 3,
+    'пятница': 4,
+    'суббота': 5,
+    'воскресенье': 6
+}
+
+DATE = namedtuple('DATE', 'day month year')
+
+def get_date(text):
+    try:
+        num_week, week_day, months = text.split()
+        num_week = int(num_week.split('-')[0])
+        week_day = WEEKDAYS[week_day]
+        months = MONTHS[months]
+    except ValueError as e:
+        logger.error(f'Вызывается ошибка ValueError: {e}')
+
+    except KeyError as e:
+        logger.error(f'Вызывается ошибка KeyError: {e}')
+
+    else:

```

Comment:
Алена, во-первых, обычно конструкцией `try-except` обрабатывают только тe строки, в которой предположительно, может появляться ошибка. В этих 4-х строках могут могут появиться две ошибки это ValueError и KeyError. Поэтому нужно обрабатывать те строки, где появляются соответствующий тип ошибки.  Кроме того, запись в лог-файл не очень информативна
```suggestion
        num_week, week_day, months = text.split()
    except ValueError:
        logger.error(f'Количество слов строки {text} – 2, ожидалось 3')
        return None
    try:
        num_week = int(num_week.split('-')[0])
    except ValueError:
        logger.error(f'Невозможно преобразовать {num_week} к целому числу')
        return None
    try:
        week_day = WEEKDAYS[week_day]
    except KeyError:
        logger.error(f'Нет такого дня недели – {week_day}')
        return None
    try:
        months = MONTHS[months]
    except KeyError:
        logger.error(f'Нет такого месяца – {months}')
        return None
```